templates/cmsSetup/Setup/Uninstall.php

```diff
@@ -0,0 +1,60 @@
+<?php
```

Comment:
I think this file is not relevant here. Maybe a copy paste from the other pull request?